### PR TITLE
Blog app - Add devise

### PR DESCRIPTION
**Pull request by Donard97**
**I created this PR to check if my project is as requested.** 👀 👀

_In this project, I add the devise gem to my app and I used it for the registration and login of users._ 💯 💯 

### Project requirements that are done:

- Remove current_user method in ApplicationController ✅ 
- Installed and setup devise  ✅ 
1. Can register a new user. ✔️ 
2. User logs in with a combination of email and password. ✔️ 
3. Hashed passwords should be stored in the database. ✔️ 
4. Ask for confirmation of email. ✔️ 
5. Can reset password. ✔️ 

- Modified the views in devise for registration and login to match the wireframes  ✅ 

 
 

<hr>

_Feel free to review 👀  my project and let me know if there are any essential changes 💪_

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
```
Comment (rloterh) [inline config/routes.rb:3]: - [x] Your _routes_ are **not properly configured** with the _devise_ gem, and hence your app is showing a blank screen when launched. Kindly run the command `rails routes` in your terminal to determine which routes are working correctly, and fix the _root_ route accordingly

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Donard, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Divise authentication set correctly in the User model
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (Donard97) [reply config/routes.rb:3]: Thank you for letting me know. I fixed this issue and now everything is working properly 👍

**Review by rukundoeric — APPROVED** [review]
Hi @Donard97,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**